test/unit/server-statefulset.bats

```diff
@@ -11,6 +11,27 @@ load _helpers
   [ "${actual}" = "true" ]
 }
 
+
+@test "server/StatefulSet: affinity not set with server.affinity" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -x templates/server-statefulset.yaml  \
+      --set 'server.affinity=' \
+      . | tee /dev/stderr |
+      yq 'length > 0' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
+@test "server/StatefulSet: affinity set with server.affinity as empty" {
```

Comment:
Since the other test checks what happens when `affinity` is null, it would be good to check the default case instead here. I'm happy to provide specific suggestions if you'd like.